From 3bdbf450e736ca831733fbb899e046769942c6df Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sun, 13 Mar 2022 18:32:20 +0100
Subject: [PATCH 3/5] defconfig: enable WPA-EAP-SUITE-B-192 ciphers

This was enabled in wpa_supplicant.git in commit 6cc245db1770
('Enable WPA-EAP-SUITE-B-192 ciphers'), without further explanation.

FIXME: It needs to be either upstreamed or dropped.
---
 wpa_supplicant/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/wpa_supplicant/defconfig b/wpa_supplicant/defconfig
index 07d9752..3f73282 100644
--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -634,3 +634,6 @@ CONFIG_WEP=y
 # design is still subject to change. As such, this should not yet be enabled in
 # production use.
 #CONFIG_PASN=y
+#
+CONFIG_SUITEB192=y
+
-- 
2.35.1

